src/main/webapp/app/exercises/programming/manage/update/update-components/difficulty/programming-exercise-difficulty.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="form-group">
```

Comment:
Moved to `General` section in `simpleMode`, so a component is needed for the difficulty selection